### DISCUSSION TITLE
Время в 12 часовом формате и температура в фаренгейтах.

### DISCUSSION THREAD

**Discussion by KeeperBY** [vvip-68/LedPanelWiFi#53] [Идеи и предложения]
Добрый день.
Можно ли такое реализовать?
Буду очень благодарен за добавление данного функционала в прошивку!

**Comment by vvip-68** [comment]
Теоретически можно, но это далеко не первый приоритет, ибо в России нафиг никому не нужно. Не пользуются тут этим.  А температура в фаренгейтах вам не нужна?

А, вижу в заголовке - нужна :). Сами то откуда будете?

**Reply by KeeperBY** [reply]
Сам то буду из Минска, но сейчас нахожусь в недружественной стране из-за несогласия с политикой наших «вождей»:-(

**Reply by vvip-68** [reply]
Кстати, в последней версии реализовано

**Reply by KeeperBY** [reply]
Ох, спасибо огромное! Попробую прошить, да контроллер поменять на ESP32, а то что-то у меня этот постоянно стал подвисать, самостоятельно автосмену эффектов отключать и терять связь с вайфаем( хотя гирлянда не большая 21х17

**Reply by vvip-68** [reply]
В последней версии сделано много оптимизаций по памяти - возможно и на 8266 будет работать получше. Хотя esp32, конечно, значительно больше памяти. Там не должно по идее падать...

Про автосмену самостоятельно - если с новой версией повторится - подробнее напишите что и когда происходит
